src/components/PayPalButtons.test.tsx

```diff
@@ -408,13 +410,15 @@ describe("<PayPalButtons />", () => {
         const mockRender = jest
             .fn()
             .mockRejectedValue(new Error("Unknown error"));
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        window.paypal!.Buttons = () => {
-            return {
-                close: jest.fn().mockResolvedValue({}),
-                isEligible: jest.fn().mockReturnValue(true),
-                render: mockRender,
-            };
+        window.paypal = {
```

Comment:
Can you try this syntax instead of the arrow functions?
```suggestion
        window.paypal = {
            Buttons() {
                return {
                    close: jest.fn().mockResolvedValue({}),
                    isEligible: jest.fn().mockReturnValue(true),
                    render: mockRender,
                };
            }
        }
```